default/python/AI/turn_state/__init__.py

```diff
@@ -0,0 +1,39 @@
+"""
+This module contains tools to work with pre-aggregated information about AI state.
+This information is not stored in the save game and recalculated each turn.
```

Comment:
a bit unclear... "is not (stored in the same game and recalculated)" or "is not (stored in the save game), but rather is recaulcated"